receiver/otlpreceiver/logs/otlp.go

```diff
@@ -76,9 +76,10 @@ func (r *Receiver) sendToNextConsumer(ctx context.Context, ld pdata.Logs) error
 		ctx = client.NewContext(ctx, c)
 	}
 
-	ctx = obsreport.StartLogsReceiveOp(ctx, r.id, receiverTransport)
+	rec := obsreport.NewReceiver(obsreport.ReceiverSettings{ReceiverID: r.id, Transport: receiverTransport})
```

Comment:
This can be cached in the receiver struct.

Reply:
Done in latest commit